test/e2e/framework/framework.go

```diff
@@ -66,7 +66,8 @@ type Framework struct {
 	BaseName string
 
 	// ClientSet uses internal objects, you should use ClientSet where possible.
-	ClientSet clientset.Interface
+	ClientSet                clientset.Interface
+	ExternalClusterClientSet clientset.Interface
```

Comment:
KubemarkExternalClusterClientSet - I know it's long, but I'd rather keep it explicit that it's for Kubemark.